biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/report/ReportBuilder.scala

```diff
@@ -395,8 +396,29 @@ trait ReportBuilder extends ToolCommand {
 
 object ReportBuilder {
 
+  /** Limiting the number of threads and instantiated futures */
+  def maxThreads = Some(2)
+  val numWorkers: Int = maxThreads.getOrElse(2)
+  val queueCapacity = 100
+
   implicit lazy val ec: ExecutionContextExecutor = ExecutionContext.global
 
+  /* TODO enable new queueing execution context when restructuring of report section is complete.
```

Comment:
This is fine but we should not forgot about this when merging this branch to develop